client/shaders/nodes_shader/opengl_vertex.glsl

```diff
@@ -175,6 +175,9 @@ float disp_z;
 	v.z = dot(eyeVec, normal);
 	tsEyeVec = normalize (v);
 
+#if (MATERIAL_TYPE == TILE_MATERIAL_PLAIN) || (MATERIAL_TYPE == TILE_MATERIAL_PLAIN_ALPHA)
+	gl_FrontColor = gl_BackColor = vec4(1.0);
```

Comment:
Entities do not actually use this shader do they?
Just wondering, I am not suggesting removing it.

Reply:
I started to propagate the "shaded" flag to nodes and changed my mind, since they are shaded by default and no one complained. I should revert this.